planner/cardinality/row_count_index.go

```diff
@@ -60,9 +60,7 @@ func GetRowCountByIndexRanges(sctx sessionctx.Context, coll *statistics.HistColl
 		}
 	}
 	recordUsedItemStatsStatus(sctx, idx, coll.PhysicalID, idxID)
-	// For the mv index case, now we have supported collecting stats and async loading stats, but sync loading and
-	// estimation is not well-supported, so we keep mv index using pseudo estimation for this period of time.
-	if !ok || idx.IsInvalid(sctx, coll.Pseudo) || isMVIndex {
+	if statistics.IndexStatsIsInvalid(idx, sctx, coll, idxID) || isMVIndex {
 		colsLen := -1
```

Comment:
Why is the comment removed?

Reply:
Should be mistake when resolving merging conflicts